README.md

```diff
@@ -6,7 +6,106 @@ The Microsoft Graph Communications API allows developers to programmatically int
+
+* If you have Visual Studio Enterprise Subscription, you can [Activate office 365] (https://my.visualstudio.com/)
+
+# Samples
+
+The Graph Calling Bot Samples are split into 2
+categories: Remote Media Bot Samples and Local Media Bot Samples.
+
+## Local media samples
+
+Local media samples gives direct access to the inbound and outbound media streams.  
+
+### [AudioVideoPlaybackBot](Samples\V1.0Samples\LocalMediaSamples\AudioVideoPlaybackBot\README.md)
+
+The AudioVideoPlaybackBot demostrates several features of local media scenarios:
+- Plays a movie in multiple resolutions as the main video output feed.
+- Listens to dominant speaker events and subscribes to inbound video feeds of those participants.
+- Allows switching between screen viewing sharer and viewer, and publishes video through the screen sharing socket.
+
+To run the sample, open [Samples\V1.0Samples\LocalMediaSamples\AudioVideoPlaybackBot.sln](Samples\V1.0Samples\LocalMediaSamples\AudioVideoPlaybackBot.sln)
+
+### [HueBot](Samples\V1.0Samples\LocalMediaSamples\HueBot\README.md)
+
+The HueBot demonstrates local media scenarios.
+- Listens to dominant speaker events and changes the hue color of the dominant speaker video.
+
+To run the sample, open [Samples\V1.0Samples\LocalMediaSamples\HueBot.sln](Samples\V1.0Samples\LocalMediaSamples\HueBot.sln)
+
+### [Policy Recording bot sample](Samples\V1.0Samples\LocalMediaSamples\PolicyRecordingBot\README.md)
+This sample demonstrates how a bot can receive media streams for recording. Please note that the sample does not actually record. This logic is left up to the developer.The system will load the bot and join it to appropriate calls and meetings in order for the bot to enforce compliance with the administrative set policy.
+
+To run the sample, open [Samples\V1.0Samples\LocalMediaSamples\PolicyRecordingBot.sln](Samples\V1.0Samples\LocalMediaSamples\PolicyRecordingBot.sln)
+
+## Remote Media Graph Calling Bot Samples
+
+### [Incident Bot Sample](Samples\V1.0Samples\RemoteMediaSamples\README.md)
+
+The Incident Bot sample is a Remote Media sample demonstrating a simple incident process workflow started by a Calling Bot.  When an incident is raised (through a custom Web API call to the bot or some other trigger), the bot will join a pre-existing Teams meeting, voice call incident responder team members via Microsoft Teams, play an audio prompt about the incident, and connect users to the incident meeting after having the callers press "1". The whole process is kicked off by a Web API call that passes the scheduled Teams meeting information and incident responders Azure AD Identities. The sample also supports incoming direct voice calls to the bot.
+
+To run the sample, open [Samples\V1.0Samples\RemoteMediaSamples\IncidentBot.sln](Samples\V1.0Samples\RemoteMediaSamples\IncidentBot.sln)
+
+## Stateless Samples
+
```

Comment:
Add a short explanation of what the Stateless Samples do - similar to the one at the start of the section on Local Media Samples.